ee/EAP7-1114_JakartaEE8.adoc

```diff
@@ -0,0 +1,42 @@
+= Pass the Jakarta EE 8 certification testsuite
+:author:            Scott Marlow
+:email:             smarlow@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+We will run and pass the Jakarta EE 8 CTS with WildFly and our EAP product.  This will be done with https://github.com/eclipse-ee4j/jakartaee-tck.  FYI, the TCK development mailing list is https://www.eclipse.org/lists/jakartaee-tck-dev/maillist.html.
+
```

Comment:
Do we have any list we will use for coordination.  As soon as we have the base configurations in place I would like to get local runs going so I can monitor changes as they happen in related to the security sections of the TCK.